catalog-items.html

```diff
@@ -378,12 +382,14 @@ <h4 class="documentation-heading">COMPANY</h4>
                         </ul>
                 </nav>
 
-                <form action="/user-email-newsletter" method="POST" class="subscription-form">   
-                    <label for="email" class="subscription-form-heading">SUBSCRIBE</label>
```

Comment:
Why did you removed this label? It is useful for screen readers. And it should be there by design.